doc/RecordingControl.xml

```diff
@@ -2200,6 +2200,61 @@ secfrac = "." 1*6DIGIT</programlisting>
         </tgroup>
       </table>
     </section>
+    <section xml:id="_refCloudRecordingObjectExamples">
+      <title>Examples</title>
+      <para>
+        Given the following storage configuration in a recording configuration:
+      </para>
+      <programlisting><![CDATA[<tt:Target>
+  <tt:Storage>1</tt:Storage>
+  <tt:Format>MP4</tt:Format>
+  <tt:Prefix>site-2</tt:Prefix>
+  <tt:Postfix>camera-5</tt:Postfix>
+  <tt:SegmentDuration>PT60S</tt:SegmentDuration>
+</tt:Target>]]></programlisting>
+      <para>
+        The key of a segment object starting on 2022-11-08 at 15:08:23.547 (UTC) is:
+      </para>
+      <programlisting>site-2/2022-11-08/camera-5/15/08-23.547Z.0.mp4</programlisting>
+      <para>
+        The key of a span end object for a span consisting of 85 segments and ending on 2022-11-08 at 16:34:11.421 (UTC) is:
+      </para>
+      <programlisting>site-2/2022-11-08/camera-5/16/34-11.421Z.85.mp4_end</programlisting>
+      <para>
+        Given the following storage configuration in a recording configuration:
+      </para>
+      <programlisting><![CDATA[<tt:Target>
+  <tt:Storage>1</tt:Storage>
+  <tt:Format>CMAF</tt:Format>
+  <tt:Prefix>site-2</tt:Prefix>
+  <tt:Postfix>camera-5</tt:Postfix>
+  <tt:SegmentDuration>PT5S</tt:SegmentDuration>
+</tt:Target>]]></programlisting>
+      <para>
+        The key of a video segment object containing starting on 2022-11-08 at 15:08:23.547 (UTC) is:
+      </para>
+      <programlisting>site-2/2022-11-08/camera-5/15/08-23.547Z.0.m4v</programlisting>
+      <para>
+        The key of a span end object for a video span consisting of 1020 segments and ending on 2022-11-08 at 16:34:11.421 (UTC) is:
+      </para>
+      <programlisting>site-2/2022-11-08/camera-5/16/34-11.421Z.85.m4v_end</programlisting>
```

Comment:
if the span consisting of 1020 segments, I am thinking span end should have segment counter 1020.

i.e 0 -> 1019 m4v segment files, end file have segment counter 1020.

site-2/2022-11-08/camera-5/16/34-11.421Z.85.m4v_end ->site-2/2022-11-08/camera-5/16/34-11.421Z.1020.m4v_end

same with m4a_end, m4m_end files.

Reply:
Thanks for the hint, classical copy-and-paste error. Fixed.